src/java/org/apache/cassandra/db/SystemKeyspace.java

```diff
@@ -1327,30 +1327,15 @@ public static void clearEstimates(String keyspace, String table)
     }
 
     /**
-     * Clears size estimates for a keyspace (used to manually clean when we miss a keyspace drop)
+     * truncates size_estimates and table_estimates tabless
      */
-    public static void clearEstimates(String keyspace)
+    public static void truncateSizeEstimates()
```

Comment:
`clearAllEstimates()` would be more intuitive to me as a method name, and inline with its sibling method.